cmd/harmony/default.go

```diff
@@ -120,6 +127,11 @@ func getDefaultConsensusConfigCopy() consensusConfig {
 	return config
 }
 
+func getDefaultSysConfigCopy() sysConfig {
```

Comment:
I was using these `getDefaultConsensusConfigCopy` functions to get default value for optional config fields (fields are pointers in harmonyConfig). I think currently `sysConfig` is a mandatory field in harmony config, so maybe we don't need this function?